### PR TITLE
Moved osg-flock docs to IDTOKEN, added system requirements.  (SOFTWARE-4548)

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

File: docs/submit/osg-flock.md [hunk 1]
```diff
@@ -38,32 +38,22 @@ and configure it only to forward jobs to the OSG.
...
-Send email to <help@opensciencegrid.org> saying that you would like to flock jobs to OSG,
-and we will [consult](#scheduling-a-planning-consultation) with you about the proper solution for your site and what
-needs to be set up at both the OSG and at your site.
+System Requirements
+-------------------
+The hardware requirement for an OSG access point depends on several factors such as number of users, 
+number of jobs and for example how I/O intensive those jobs are. Our minimum recommended configuration
+is 6 cores, 12 GB RAM and 1 TB of local disk. The hardware can be bare metal or virtual machine, but
+we do not recommend containers as these submit host are running a bunch of system services which can
+be difficult to configure in a container.
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:47]:
```suggestion
The hardware requirement for an OSG access point depends on several factors such as number of users,
number of jobs and for example how I/O intensity of those jobs.
Our minimum recommended configuration is 6 cores, 12 GB RAM and 1 TB of local disk.
The hardware can be bare metal or virtual machine, but we do not recommend containers as these submit host are running
many system services which can be difficult to configure in a container.
```

Minor wording changes and line splits to try to reduce future diffs: https://opensciencegrid.org/technology/documentation/style-guide/#meta

File: docs/submit/osg-flock.md [hunk 2]
```diff
@@ -38,32 +38,22 @@ and configure it only to forward jobs to the OSG.
...
 * __Network:__ 
     * Inbound TCP port 9618 must be open.
     * The access point must have a public IP address with both forward and reverse DNS configured.
 
-As with all OSG software installations, there are some one-time steps to prepare in advance:
-
-* Obtain root access to the host
-* Prepare [the required Yum repositories](../common/yum.md)
-* Install [CA certificates](../common/ca.md)
-
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:66]: I'd prefer to keep the first two bullets here. For the third bullet, is `osg-flock` configured to use system CAs and do we not need to worry about IGTF CAs?

File: docs/submit/osg-flock.md [hunk 3]
```diff
@@ -83,26 +73,21 @@ You will need information like the hostname, and the administrative and security
...
 
 
 Installing Required Software
 ----------------------------
 Flocking requires HTCondor software as well as software for reporting to the OSG accounting system.
-OSG provides a convenience RPM that installs all required packages.
-To install, run:
+Start by setting up the OSG YUM repositories following the
+[Installing Yum Repositories](../common/yum.md). __Note that you have to use OSG 3.6__. Earlier
+versions will not work.
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:87]: Why is this the case? We ship 8.9.11 in 3.5 upcoming. I'm a little hesitant to push 3.6 right now but as long as we know that this works, I'm ok with telling folks to use 3.6

File: docs/submit/osg-flock.md [hunk 4]
```diff
@@ -146,31 +131,24 @@ we provide a recommended configuration for flocking.
 
 Configuring Authentication
 --------------------------
-If you have obtained a host certificate according to the [instructions](../security/host-certs.md),
-you do not need to do any additional configuration and may [continue to the next section](#managing-services).
+Create a file named `/etc/condor/tokens.d/flock.opensciencegrid.org` with the IDTOKEN you
+received earlier. The token you only take up one line (that is, there should not be any
+line breaks in the token).
```
Comment (brianhlin) [inline docs/submit/osg-flock.md:136]:
```suggestion
Create a file named `/etc/condor/tokens.d/flock.opensciencegrid.org` with the IDTOKEN you received earlier.
Ensure that there aren't any line breaks in this file (i.e., the entire token should only take up one line).
```  

Wording/spacing suggestion

**Review by brianhlin — COMMENTED** [review]
Some suggestions and questions

**Comment by brianhlin** [comment]
Hey @rynge, any thoughts on my questions/comments above?

File: docs/submit/osg-flock.md (hunk 3 above)
Comment (rynge) [reply docs/submit/osg-flock.md:87]: 3.6 is better long term here. That is what I did the testing with.

File: docs/submit/osg-flock.md (hunk 2 above)
Comment (rynge) [reply docs/submit/osg-flock.md:66]: We do not need CAs anymore - these are just straight tokens. The Yum repos install is in the next paragraph.

File: docs/submit/osg-flock.md (hunk 3 above)
Comment (brianhlin) [reply docs/submit/osg-flock.md:87]: Ok sure. Could you make this note an [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) to highlight it further?

File: docs/submit/osg-flock.md (hunk 2 above)
Comment (brianhlin) [reply docs/submit/osg-flock.md:66]: IIRC there was one last place that GSI was used in the GlideinWMS inf. Was that between factory and FE?

File: docs/submit/osg-flock.md (hunk 3 above)
Comment (rynge) [reply docs/submit/osg-flock.md:87]: Yes, but let's get this in place and I can take a second pass on it later

File: docs/submit/osg-flock.md (hunk 2 above)
Comment (rynge) [reply docs/submit/osg-flock.md:66]: That is only FE->factory->glidein. No GSI needed between submit hosts and CM.

**Review by brianhlin — APPROVED** [review]

**Pull request merged**